packages/vue-client/src/components/PlayingTable/ThueMorseTable.vue

```diff
@@ -1,12 +1,15 @@
 <script setup lang="ts">
-import { BadukState, GridBadukConfig } from "@ogfcommunity/variants-shared";
-import BadukBoardSelector from "../boards/BadukBoardSelector.vue";
+import {
+  DefaultBoardState,
+  NewGridBadukConfig,
+} from "@ogfcommunity/variants-shared";
 import MoveSequenceDisplay from "../MoveSequenceDisplay.vue";
 import { count_binary_ones } from "@ogfcommunity/variants-shared";
+import DefaultBoard from "../boards/DefaultBoard.vue";
 
 defineProps<{
-  gamestate: BadukState;
-  config: GridBadukConfig;
+  gamestate: DefaultBoardState;
+  config: NewGridBadukConfig;
```

Comment:
This is okay because we have `sanitizeConfig`

Reply:
Hm I think this should be `NewBadukConfig` though, or `DefaultBoardConfig`, since Thue-Morse can be played on graph boards too.

Reply:
Oops you're right, just seeing this comment. I'll follow up